src/web_ui/__init__.py

```diff
@@ -1,27 +1,38 @@
 # Set up logging
 import logging
 
-from multiprocessing.sharedctypes import Value
+from constants import NAME_KEY_MAP
 from flask import Flask, render_template, request
-
 from node_mgmt import NetworkEnv, EnvScanner, get_ssh_fingerprint, Node
 from node_mgmt.commands import (
     imt_sensor_address,
     holykell_sensor_address_7,
     holykell_sensor_address_8,
     trigger_config_generation
 )
+
```

Comment:
We could probably clean up the imports here and make sure we follow PEP-8 (see second bullet point on grouping under https://peps.python.org/pep-0008/#imports)